src/EnergyPlus/GeneratorFuelSupply.cc

```diff
@@ -268,27 +251,7 @@ namespace GeneratorFuelSupply {
         // METHODOLOGY EMPLOYED:
         // Hardcoded data from NIST is filled into data structure one time only
 
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int NumHardCodedConstituents; // number of gases included in data
-        Real64 LHVfuel;               // lower heating value of fuel, working var
-        Real64 HHVfuel;               // higher heating value of fuel, working var
-        Real64 O2Stoic;               // stochiometric oxygen coef in chemical equation (15)
-        Real64 CO2ProdStoic;          // product gases carbon dioxide coeff
-        Real64 H2OProdStoic;          // product gases water coeff
-        int i;                        // loop index
-        std::string thisName;         // working string var
-        int thisGasID;                // working index in Gas phase data structure
-        int CO2dataID;                // hard wired to CO2 index in gas data struct
-        int WaterDataID;              // hard wired to Water index in gas data struct
-        Real64 LHVi;                  // working var for lower heating value calc
-        Real64 HHVi;                  // working var for higher heating value calc
-        //  INTEGER   :: thisConstituent
-        Real64 MWfuel;
-        // unused  REAL(r64) :: DelfHfuel
-        // unused  REAL(r64) :: h_i
-        // unused  REAL(r64) :: LHV
```

Comment:
[src/EnergyPlus/GeneratorFuelSupply.cc:278]:(style),[variableScope],The scope of the variable 'i' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:279]:(style),[variableScope],The scope of the variable 'thisName' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:280]:(style),[variableScope],The scope of the variable 'thisGasID' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:281]:(style),[variableScope],The scope of the variable 'CO2dataID' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:282]:(style),[variableScope],The scope of the variable 'WaterDataID' can be reduced.